test/mbedls_toolsbase.py

```diff
@@ -176,16 +181,24 @@ def test_fs_after(self):
         with patch("mbed_lstools.lstools_base.MbedLsToolsBase._read_htm_ids") as _read_htm:
             new_device_id = "00017531642046"
             _read_htm.return_value = (new_device_id, {})
-            filter = re.compile("")
-            ret = self.base._fs_after_id_check(device, filter)
+            filter = None
+            ret = self.base._fs_after_id_check(deepcopy(device), filter)
             self.assertIsNotNone(ret)
             self.assertEqual(ret['target_id'], new_device_id)
             _read_htm.assert_called_with(device['mount_point'])
 
             _read_htm.reset_mock()
 
-            filter_out = re.compile("NOT-K64F")
-            ret = self.base._fs_after_id_check(device, filter_out)
+            filter_in = lambda m: m['target_id'] == device['target_id_usb_id']
```

Comment:
Propose to add this as example for public api doc string

Reply:
Sure, can do